src/test/java/uk/ac/ebi/embl/api/validation/submission/SubmissionValidationPlanTest.java

```diff
@@ -479,4 +481,25 @@ public void testGenomeSubmissionwitFastawithValidChromosomeListMultipleTimesInPa
 
 		CompletableFuture.allOf(futs.toArray(new CompletableFuture[futs.size()])).join();
 	}
+
```

Comment:
if possible, can you please add a valid case and assert the expected and actual sequence length are same.